com.w1hkj.fldigi.yml

```diff
@@ -0,0 +1,52 @@
+app-id: com.w1hkj.fldigi
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: fldigi
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=all
+  - --share=network
+  - --socket=pulseaudio
+
```

Comment:
You should add a `cleanup` list, with at least `*.a` and `/include`.

Reply:
ok